### ISSUE TITLE
'import()' expressions are not supported yet inconsistency

### ISSUE THREAD

**Issue by electriquo** [mysticatea/eslint-plugin-node#349]
When running ESLint, I get
```shell
error  'import()' expressions are not supported yet  node/no-unsupported-features/es-syntax
```
Where `import()` is [compatible since Node.js 13.2.0 ](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import#browser_compatibility) and I am using Node.js 18.15.0

```shell
$ cat package.json | jq '.engines'
{
  "node": ">=18.15.0",
  "npm": ">=9.6.0"
}
```

The error seems to be false negative.
Is this a bug or did is my configuration the issue?

**Comment by robatwilliams** [comment]
I have what seems to be the same issue - "Import and export declarations are not supported yet". Using v11.1.1 of this plugin.

Engines is `"node": "^14.18.0 || >=16.0.0"`.

Even if I put an override comment at the top of a specific file to configure the version, I still get the error.

**Comment by robatwilliams** [comment]
There's a big map of features and versions supported since here. There is `null` in it for this particular feature.

https://github.com/mysticatea/eslint-plugin-node/blob/master/lib/rules/no-unsupported-features.js#L263

**Comment by voxpelli** [comment]
[`eslint-plugin-n`](https://github.com/eslint-community/eslint-plugin-n) is the maintained version of this module.

We switched to it in eg. [`eslint-config-standard`](https://github.com/standard/eslint-config-standard) / [`standard`](https://github.com/standard/standard) and it is maintained by me and other members of the official [ESLint community](https://github.com/eslint-community) organization.

**Comment by voxpelli** [comment]
@foolioo exactly ;)
